pages/resources/index.vue

```diff
@@ -188,6 +188,17 @@ export default Vue.extend<Data, Methods, Computed, never>({
   watch: {
     '$route.query': function() {
       this.fetchResults()
+    },
+
+    currentResourceCount: {
```

Comment:
Can `resourceHeading` be [a computed value instead of being set in a watcher](https://vuejs.org/v2/guide/computed.html#Computed-vs-Watched-Property)?